browser/models.py

```diff
@@ -322,6 +322,7 @@ class Investigator(YearMixin, CuratedMixin, LastUpdatedMixin):
     person = models.ForeignKey('Person')
     subject = models.CharField(max_length=255, blank=True)
     role = models.CharField(max_length=255, blank=True)
+    institution = models.CharField(max_length=255, blank=True)
```

Comment:
isn't there an institution class?

Reply:
Yes, there is an institution class. I have added institution model to the institution field.